crates/cli/src/subcommands/publish.rs

```diff
@@ -163,6 +167,30 @@ pub async fn exec(mut config: Config, args: &ArgMatches) -> Result<(), anyhow::E
         database_host
     );
 
+    if delete_tables {
+        if force {
+            eprintln!("Skipping confirmation due to --force.");
+        } else {
+            // Note: `name_or_address` should be set, because it is `required` in the CLI arg config.
+            println!(
+                "This will DESTROY the current {} module, and ALL corresponding data.",
+                name_or_address.unwrap()
+            );
+            print!(
+                "Are you sure you want to proceed? (y/N) [deleting {}] ",
+                name_or_address.unwrap()
+            );
+            std::io::stdout().flush()?;
+            let mut input = String::new();
+            std::io::stdin().read_line(&mut input)?;
+            if input.trim().to_lowercase() != "y" && input.trim().to_lowercase() != "yes" {
+                println!("Aborting");
+                return Ok(());
+            }
+        }
+        query_params.push(("clear", "true"));
+    }
+
     eprintln!("Publishing module...");
```

Comment:
Even though its not part of this PR I would recommend changing this to `println` as well